src/test/datascience/mountedWebView.ts

```diff
@@ -59,6 +59,9 @@ export class MountedWebView implements IMountedWebView, IDisposable {
     private visible = true;
     private disposedEvent = new EventEmitter<void>();
     private loadFailedEmitter = new EventEmitter<void>();
+    public get viewColumn() {
+        return 1;
```

Comment:
I'd use the VS Code enums instead of `1`

Reply:
This is test infra and may not be running in vscode, so I thought it would be better to use `1` here.